src/IssueViz.Security/Taint/TaintList/TaintVulnerabilitiesControl.xaml

```diff
@@ -1,15 +1,124 @@
 ﻿<UserControl x:Class="SonarLint.VisualStudio.IssueVisualization.Security.Taint.TaintList.TaintVulnerabilitiesControl"
              x:ClassModifier="internal"
              xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:vsCatalog="clr-namespace:Microsoft.VisualStudio.Imaging;assembly=Microsoft.VisualStudio.ImageCatalog"
+             xmlns:vsImaging="clr-namespace:Microsoft.VisualStudio.Imaging;assembly=Microsoft.VisualStudio.Imaging"
+             xmlns:vsTheming="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Shell.14.0"
+             DataContext="{Binding ViewModel, RelativeSource={RelativeSource Mode=Self}}"
+             >
     <UserControl.Resources>
         <ResourceDictionary>
             <ResourceDictionary.MergedDictionaries>
                 <ResourceDictionary Source="../../SharedUI/SharedResources.xaml" />
             </ResourceDictionary.MergedDictionaries>
+
+            <!-- Extend the shared data grid row style to add the context menu and navigability data bindings -->
+            <Style BasedOn="{StaticResource BaseDataGridRow}"
+                       TargetType="DataGridRow">
+                <Setter Property="Background" Value="{DynamicResource {x:Static vsTheming:TreeViewColors.BackgroundBrushKey}}"/>
```

Comment:
FYI setting the background here is the same hack as in the hotspot control.

I haven't worked out why it's required, but I do know a bit more context: the `DataGridCell` has `<Setter Property="Padding" Value="6,2,2,2"/>`.
This means the data cell isn't filling the whole cell, which is exposing part of the control behind it, which is a border that is a child of the `DataGridRow`. For some reason, the border isn't picking up the background set for its parent unless it's set here.